src/sentry/models/commitfilechange.py

```diff
@@ -40,3 +41,23 @@ class Meta:
     @staticmethod
     def is_valid_type(value: str) -> bool:
         return value in COMMIT_FILE_CHANGE_TYPES
+
+
+def process_resource_change(instance, **kwargs):
+    from sentry.integrations.github.integration import GitHubIntegration
+    from sentry.integrations.gitlab.integration import GitlabIntegration
+    from sentry.tasks.codeowners import code_owners_auto_sync
```

Comment:
Should this be something that the integration installation abstractions provide? Something like `install.codeowners_locations`. Repository integrations could return an empty list by default.

Reply:
@markstory hmm CODEOWNERS is only supported by GitHub/GitLab, so it's only `codeowners_locations` is only relevant to those integrations. I don't think it's common enough to move into an abstraction. Also I don't want to do db lookups to find the integration installation before triggering a task.